cluster-provision/gocli/cmd/run.go

```diff
@@ -379,7 +379,8 @@ func run(cmd *cobra.Command, args []string) (err error) {
 
 		for i := 0; i < int(secondaryNics); i++ {
 			netSuffix := fmt.Sprintf("%d-%d", x, i)
-			nodeQemuArgs = fmt.Sprintf("%s -device virtio-net-pci,netdev=secondarynet%s -netdev tap,id=secondarynet%s,ifname=stap%s,script=no,downscript=no", nodeQemuArgs, netSuffix, netSuffix, netSuffix)
+			macSuffix := fmt.Sprintf("%d%d", x, i)
```

Comment:
I don't know if we'd ever have a reason to have more than 10 nodes, but it would be pretty easy to make this a little more robust just in case by doing something like what I have below if you wanted.

```
macCounter := 0
...
         macSuffix := fmt.Sprintf("%02x", macCounter)
         macCounter++
```

Reply:
Done, thanks!